packages/wdio-config/tests/configparser.test.js

```diff
@@ -102,7 +102,7 @@ describe('ConfigParser', () => {
             const specs = configParser.getSpecs()
             expect(specs).toContain(__filename)
             expect(specs).toContain(INDEX_PATH)
-            expect(specs).toContain(path.join(__dirname, 'validateConfig.test.js'))
+            expect(specs).not.toContain(path.join(__dirname, 'validateConfig.test.js'))
```

Comment:
Why do we need to invert this assertion?

Reply:
The original assertion was not actually checking the _exclude_ defined within FIXTURES_CONF which has:
```
    exclude: [
        path.join(TEST_ROOT, '/validateConfig.test.js')
    ]
```